tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -576,6 +576,21 @@ describe(__filename, () => {
     expect(root.find(FlagReviewMenu)).toHaveLength(0);
   });
 
+  it('hides the flag button if you wrote the review', () => {
+    const originalReviewId = 123;
+    const review = signInAndDispatchSavedReview({
+      siteUserId: originalReviewId,
```

Comment:
This variable name is confusing but I see what you are trying to do. You are letting the helper take over to set `siteUserId` and `reviewUserId` to the same value. Instead of letting the helper do this implicitly, your test should explicitly set these both to the same value so that the intention is clear.

Reply:
Yeah, I didn't want to introduce another helper that wrapped the `signInAndDispatchSavedReview()` helper so I just used it directly. I've updated the variable naming to `siteUserId` so it hopefully helps us recognize that this is an authenticated user.